api/nodes/views.py

```diff
@@ -293,6 +305,8 @@ def perform_create(self, serializer):
     # overrides BulkDestroyJSONAPIView
     def allow_bulk_destroy_resources(self, user, resource_list):
         """User must have admin permissions to delete nodes."""
+        if self.request.query_params.get('skip_uneditable', 'False') == 'True':
+            return True
```

Comment:
This will return `True` if a user requests a set in which they have no write access for any object. Consider rewrite:
```python
if utils.is_truthy(self.request.query_params.get('skip_uneditable', 'False')):
    return any([node.has_permission(user, ADMIN) for node in resource_list])